richtextfx/src/integrationTest/java/org/fxmisc/richtext/api/FxmlTester.java

```diff
@@ -0,0 +1,26 @@
+package org.fxmisc.richtext.api;
+
+import javafx.stage.Stage;
+import javafx.fxml.FXMLLoader;
+import org.fxmisc.richtext.RichTextFXTestBase;
+import org.junit.Test;
+
+public class FxmlTester extends RichTextFXTestBase {
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        // Nothing needed here
+    }
+
+    @Test
+    public void test_fxml_construction_of_area()
+    {
+        Object obj = null;
+        // FxmlTest.fxml is located in resources folder:
+        // src/integrationTest/resources/org/fxmisc/richtext/api/
+        try	{ obj = FXMLLoader.load( getClass().getResource( "FxmlTest.fxml" ) ); }

```

Comment:
This should return an area (you'll need to cast the returned object into what it actually is) so we can assert that its properties are what they were specified to be in the FXML document.